storage_image_product_brand/views/product_brand_image_relation.xml

```diff
@@ -14,6 +14,12 @@
             <form position="attributes">
                 <attribute name="string">Images</attribute>
             </form>
+            <group name="extra" position="inside">
+                <field
+                    name="tag_id"
+                    context="{'active_model': context.get('active_model')}"
```

Comment:
When I saw the same code in the other modules i wondered: is this really necessary?
If `active_model` is already in context, why re-setting it?

Reply:
In fact the field tag_id is doesn't not exist in the abstract model and not in the abstract view.
Maybe we can add it by default ?

Reply:
I just realise that I didn't understand you question.
Yes it's needed, indeed odoo do not propagate the context by default for the many2one during the quick create, so we need to pass it explicitly. Just tested right now

Reply:
Good to know 👍 thanks!

Reply:
good to know, indeed :smile: :pray: